app.py

```diff
@@ -55,6 +87,58 @@ def choose_game_type_handler(handler_input: HandlerInput) -> Response:
     return handler_input.response_builder.response
 
 
+@sb.request_handler(can_handle_func=lambda input:
+                    is_currently_playing(input) and
+                    is_intent_name('NumberGuessIntent')(input))
+def number_guess_handler(handler_input):
+    """Handler for processing guess with target."""
+    # type: (HandlerInput) -> Response
+    session_attr = handler_input.attributes_manager.session_attributes
+    target_num = eval(session_attr['lastQuestionAsked'][0]
+                      + session_attr['operator']
+                      + session_attr['lastQuestionAsked'][1]
+                      )
+    
+    guess_num = int(handler_input.request_envelope.request.intent.slots[
+        'number'].value)
+    
+    # check answer.
+    if guess_num == target_num:
+        speech_text = (
+            f'That\'s correct!'
```

Comment:
doesn't need to be an f-string. Just make this `speech_text = 'That\'s correct!'` on a single line.